src/System.IO.Pipes/tests/project.json

```diff
@@ -15,6 +15,7 @@
     "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-rc3-24026-00",
     "System.Security.Principal": "4.0.1-rc3-24026-00",
     "System.Text.RegularExpressions": "4.1.0-rc3-24026-00",
+    "System.Threading.Overlapped": "4.0.1-rc3-24026-00",
```

Comment:
Is throwing the exception basically a way to get around rid validation until we get rid-specific dependencies, or is there more to this?